package.json

```diff
@@ -11,6 +11,7 @@
   "dependencies": {
     "classnames": "^2.2.1",
     "d.js": "^0.7.3",
+    "flexboxgrid": "^6.3.0",
```

Comment:
No, I added it in meantime because for some reason I had a problem with this dependency